src/encoder/ffmpeg.cpp

```diff
@@ -200,4 +199,9 @@ std::vector<Packet> Codec::encode(const shar::Frame& image) {
   return packets;
 }
 
+// PTS calculation formula: (1 / FPS) * sample rate * frame number
+int Codec::get_pts() {
+  return static_cast<int>((1 / static_cast<double>(m_context->time_base.den)) * CLOCK_RATE * m_frame_counter++);
```

Comment:
I think we can avoid using floating point arithmetic here. `CLOCK_RAGE/fps` should provide good enough approximation in most cases.